### PR TITLE
[W6.1e][T09-B3]Aditya Agarwal

### PR DESCRIPTION
**Pull request by AdityaA1998**

### CODE REVIEW

**Comment by AdityaA1998** [comment]
Hey Jeffry, 

Check my updated commit as I had already made a pull request before but I was asked to update the method name for inAnyNull() test and I did that during the class and commited. 

Thank you

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -36,11 +35,48 @@ public void elementsAreUnique() throws Exception {
...
     private void assertAreUnique(Object... objects) {
         assertTrue(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
 
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+    
+    
+    private void isAnyNull_EmptyList_returnsFalse() {
```
Comment (jeffryhartanto) [inline test/java/seedu/addressbook/common/UtilsTest.java:54]: Can add the annotation @Test

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -36,11 +35,48 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() throws Exception {
+        isAnyNull_EmptyList_returnsFalse();
+        isAnyNull_NoNull_returnsFalse();
+        isAnyNull_FewNull_returnsTrue();
```
Comment (jeffryhartanto) [inline test/java/seedu/addressbook/common/UtilsTest.java:42]: Given these methods, you can remove `isAnyNull` method.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 3]
```diff
@@ -36,11 +35,48 @@ public void elementsAreUnique() throws Exception {
...
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+    
+    
+    private void isAnyNull_EmptyList_returnsFalse() {
+        // empty list
+        assertNoNullObject();
+    }
+
+    private void isAnyNull_NoNull_returnsFalse() {
```
Comment (jeffryhartanto) [inline test/java/seedu/addressbook/common/UtilsTest.java:59]: Coding standard violation.. `NoNull` should be in camelCase.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 4]
```diff
@@ -36,11 +35,48 @@ public void elementsAreUnique() throws Exception {
...
+        // no null objects
+        assertNoNullObject("abc", "abc");
+        assertNoNullObject("abc", "", "abc", "ABC");
+        assertNoNullObject("", "abc", "a", "abc");
+    }
+
+    private void isAnyNull_FewNull_returnsTrue() {
+        // few null objects
```
Comment (jeffryhartanto) [inline test/java/seedu/addressbook/common/UtilsTest.java:67]: You can remove the comment.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 5]
```diff
@@ -36,11 +35,48 @@ public void elementsAreUnique() throws Exception {
...
+    }
+
+    private void isAnyNull_NoNull_returnsFalse() {
+        // no null objects
+        assertNoNullObject("abc", "abc");
+        assertNoNullObject("abc", "", "abc", "ABC");
+        assertNoNullObject("", "abc", "a", "abc");
+    }
+
+    private void isAnyNull_FewNull_returnsTrue() {
```
Comment (jeffryhartanto) [inline test/java/seedu/addressbook/common/UtilsTest.java:66]: `hasNull` might be a better name.

**Comment by jeffryhartanto** [comment]
@AdityaA1998 Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**